### ISSUE TITLE
empty repo

### ISSUE THREAD

**Issue by anoymouserver** [hopglass/hopglass#2]
Is there any reason why this repository exists, but the actual code is still at [plumpudding/hopglass](/plumpudding/hopglass)?

**Comment by plumpudding** [comment]
The plan was to port @xf-'s code to the original meshviewer, make a PR and rebase HopGlass onto that. The resulting place should've been placed here.

**Comment by xf-** [comment]
yeah code is still available (and i'm still follow all changes),. but i think this repo can be removed.
https://github.com/ffrgb/meshviewer

**Comment by SimJoSt** [comment]
Are the plans to "merge" @xf- meshviewer fork with the hopglass frontend still active?
In Bremen we have 3 people working on frontend and backend of meshviewer functionalities and they would like to avoid as much duplicate work as possible and would want to push any upcoming code upstream. Using a fork of meshviewer or hopglass isn't the preferred solution.

**Comment by plumpudding** [comment]
@SimJoSt yes the plans still exist but I'm not actively working on them. Do a PR to plumpudding/hopglass or better create an issue to discuss if the functionality should be included in HopGlass.
